proto/provenance/metadata/v0/contract.proto

```diff
@@ -0,0 +1,88 @@
+syntax = "proto3";
+
+package contract;
+
+option go_package          = "github.com/provenance-io/provenance/x/metadata/legacy/v039";
+option java_package        = "io.provenance.metadata.v0";
+option java_multiple_files = true;
+
+import "gogoproto/gogo.proto";
+import "provenance/metadata/v0/spec.proto";
+import "provenance/metadata/v0/types.proto";
+
+import "google/protobuf/timestamp.proto";
+
+message Recital {
+  types.PartyType                      signer_role = 1;
+  types.SigningAndEncryptionPublicKeys signer      = 2;
+  bytes                                address     = 3;
```

Comment:
what's best practice in Cosmos now, bytes or strings for addresses?  No need to change, just curious.

Reply:
its supposed to be strings everywhere ... but here in `v0` proto I am trying to follow the existing Figure spec